packages/cyberstorm/package.json

```diff
@@ -28,7 +28,7 @@
     "@radix-ui/react-dialog": "^1.0.2",
     "@radix-ui/react-dropdown-menu": "^2.0.1",
     "@radix-ui/react-radio-group": "^1.1.3",
-    "@radix-ui/react-select": "^1.2.0",
+    "@radix-ui/react-select": "^2.0.0",
```

Comment:
IDK how Radix uses semver but is it wise to mix components from v2 with components from v1?

Reply:
From what I could find, it's alright to mix them.